### ISSUE TITLE
project depends on artifact from unspecified repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Lesmiscore/rcon-client#4]
```
What steps will reproduce the problem?
1. mvn package

What is the expected output? What do you see instead?

[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Failed to resolve artifact.

Missing:
----------
1) com.google.named-logger:NamedLogger:jar:0.0-SNAPSHOT

  Try downloading the file manually from the project website.

  Then, install it using the command: 
      mvn install:install-file -DgroupId=com.google.named-logger -DartifactId=NamedLogger -Dversion=0.0-SNAPSHOT -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there: 
      mvn deploy:deploy-file -DgroupId=com.google.named-logger -DartifactId=NamedLogger -Dversion=0.0-SNAPSHOT -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency: 
        1) com.google.rcon-client:RConClient:jar:1.0-SNAPSHOT
        2) com.google.named-logger:NamedLogger:jar:0.0-SNAPSHOT

----------
1 required artifact is missing.

for artifact: 
  com.google.rcon-client:RConClient:jar:1.0-SNAPSHOT

from the specified remote repositories:
  central (http://repo1.maven.org/maven2)


What version of the product are you using? On what operating system?


I checked out from SVN today (nov 21, 2012)


Please provide any additional information below.

You probably need to adjust your pom.xml to specify the repository that hosts 
the NamedLogger artifact so that regular people can compile the source.

```

Original issue reported on code.google.com by `thoth...@gmail.com` on 21 Nov 2012 at 7:14

**Comment by GoogleCodeExporter** [comment]
```
You are correct with your point. The artifact NamedLogger isn't installed in 
any public repository. It is however an other public project of mine here on 
google code https://code.google.com/p/named-logger/ . You can download and 
install it in your own reposirory and then you can compile the RConClient.

To solve this issue, I will remove the dependency of RConClient to NamedLogger. 
I had already planned it, but haven't taken the time to change this project.
```

Original comment by `Vincent....@gmail.com` on 21 Nov 2012 at 8:11
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
My favorite logging library is log4j.  I think the apache commons has another 
logging library..
```

Original comment by `thoth...@gmail.com` on 21 Nov 2012 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
I have removed the logging (and the dependencies to NamedLogger).

The issue is fixed in version 1.1.
```

Original comment by `Vincent....@gmail.com` on 21 Nov 2012 at 9:53
- Changed state: **Fixed**
